### PR TITLE
PodConfigMapsAndSecrets should handle Projected volume

### PR DESCRIPTION
**Pull request by wackxu**
Signed-off-by: <xushiwei5@huawei.com>

**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

PodConfigMapsAndSecrets should handle Projected volume

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cloud/pkg/edgecontroller/manager/location.go [hunk 1]
```diff
@@ -29,6 +29,16 @@ func (lc *LocationCache) PodConfigMapsAndSecrets(pod v1.Pod) (configMaps, secret
 		if v.Secret != nil {
 			secrets = append(secrets, v.Secret.SecretName)
 		}
+		if v.Projected != nil {
+			for _, source := range v.Projected.Sources {
+				switch {
+				case source.ConfigMap != nil:
+					configMaps = append(configMaps, source.ConfigMap.Name)
+				case source.Secret != nil:
+					secrets = append(secrets, source.Secret.Name)
```
Comment (vincentgoat) [inline cloud/pkg/edgecontroller/manager/location.go:38]: switch&case will only return one case, will there contain both source.ConfigMap.Name and source.Secret.Name?

Comment (fisherxu) [reply cloud/pkg/edgecontroller/manager/location.go:38]: Yes, in projected volume in kubelet, it also use the switch case for sources. Ref:  https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/projected/projected.go#L254

**Comment by fisherxu** [comment]
Need to rebase to the master branch to fix the failed ci :)

**Comment by wackxu** [comment]
> Need to rebase to the master branch to fix the failed ci :)

Done

**Review by fisherxu — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**